package.json

```diff
@@ -24,6 +24,12 @@
     "type": "git",
     "url": "git://github.com/js-cookie/js-cookie.git"
   },
+  "files": [
+    "src/**/*.js",
+    "MIT-LICENSE.txt",
+    "SERVER_SIDE.md",
+    "CONTRIBUTING.md"
```

Comment:
It's 01:50 AM here and because I am falling asleep I just read CHANGELOG as if it was CONTRIBUTING, hahahaha.

Nevermind that, thanks :)